packages/components/src/Combobox/Combobox.test.tsx

```diff
@@ -1,530 +1,339 @@
+      expect(screen.getByTestId(MENU_TEST_ID)).not.toHaveClass("hidden");
+    });
   });
 
-  it("should throw an error when Option/Action and Content all exist as siblings", () => {
-    const component = () =>
-      render(
-        <Combobox onSelect={mockOnSelect} selected={[]}>
-          <Combobox.TriggerButton label="Button" />
-
-          <Combobox.Content options={[]} onSelect={jest.fn()} selected={[]} />
+  it("should fire the onClick of an action when clicking an action", async () => {
+    await userEvent.click(screen.getByText(activatorLabel));
+    await userEvent.click(screen.getByText("Add Teammate"));
+    expect(handleAction).toHaveBeenCalledTimes(1);
+  });
 
-          <Combobox.Option id="1" label="Option 1" />
-          <Combobox.Option id="2" label="Option 2" />
+  it("should fire the onSelect when clicking an option", async () => {
+    await userEvent.click(screen.getByText(activatorLabel));
+    await userEvent.click(screen.getByText("Bilbo Baggins"));
 
-          <Combobox.Action label="Action 1" onClick={mockActionClick} />
-        </Combobox>,
-      );
-
-    expect(component).toThrow(COMBOBOX_OPTION_AND_CONTENT_EXISTS_ERROR);
+    expect(handleSelect).toHaveBeenCalledTimes(1);
+    expect(handleSelect).toHaveBeenCalledWith([
+      { id: "1", label: "Bilbo Baggins" },
+    ]);
   });
 });
 
-describe("ComboboxContent", () => {
-  it("should not have the content visible by default", () => {
-    const { getByTestId } = render(
-      <Combobox>
-        <Combobox.TriggerButton label="Button" />
-        <Combobox.Content
-          options={[]}
-          onSelect={jest.fn()}
-          selected={[]}
-        ></Combobox.Content>
-      </Combobox>,
-    );
-    expect(getByTestId("ATL-Combobox-Content")).toHaveClass("hidden");
+describe("Combobox Single Select", () => {
+  const selectedValue = { id: "1", label: "Bilbo Baggins" };
+  beforeEach(() => {
+    mockSelectedValue.mockReturnValueOnce([selectedValue]);
+    renderCombobox();
   });
 
-  it("should close the content after opening and making a (single) selection", () => {
-    const { getByTestId, getByText } = render(
-      <Combobox>
-        <Combobox.TriggerButton label="Click Me" />
-        <Combobox.Content
-          options={[
-            { id: "1", label: "Bilbo Baggins" },
-            { id: "2", label: "Frodo Baggins" },
-          ]}
-          onSelect={jest.fn()}
-          selected={[]}
-        ></Combobox.Content>
-      </Combobox>,
-    );
-
-    const button = getByText("Click Me");
-    fireEvent.click(button);
-
-    expect(getByTestId("ATL-Combobox-Content")).not.toHaveClass("hidden");
-
-    const option = getByText("Bilbo Baggins");
-    fireEvent.click(option);
-
-    expect(getByTestId("ATL-Combobox-Content")).toHaveClass("hidden");
+  it("should not show the heading when there's a selection", () => {
+    expect(screen.queryByText(activatorLabel)).not.toBeInTheDocument();
```

Comment:
hmmm in this case though, we do want that activator label. so I guess we do need to keep it around for assertions like this, but maybe we use the `getByRole` for finding the button VS checking the button text like this